sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -921,6 +922,15 @@ def connections(self) -> ConnectionsOperations:
         """
         return self._connections
 
+    @property
+    def workspace_connections(self) -> ConnectionsOperations:
```

Comment:
should we add a deprecation notice here too?

Reply:
Done

Reply:
Removed, apparently this was never a real thing.